doc/man3/OSSL_PARAM.pod

```diff
@@ -267,6 +267,14 @@ B<OSSL_PARAM_OCTET_STRING>), but this is in no way mandatory.
 
 =item *
 
+If I<data> for a B<OSSL_PARAM_OCTET_STRING> or a
+B<OSSL_PARAM_UTF8_STRING> is NULL, the I<responder> should
+set I<return_size> to the size of the item to be returned
+and return success. Later the responder will be called again
```

Comment:
Should this just say 1 instead of success.
Not sure the later part is required.

Reply:
Will it always be 1? I tried to follow the language in the rest of the document which talks about success and errors rather than 1 or 0. Later sentence could be removed but I think it does clarify: if you feel strongly I can take it out.